spring-cloud-deployer-resource-support/src/main/java/org/springframework/cloud/deployer/resource/support/DelegatingResourceLoader.java

```diff
@@ -137,12 +137,10 @@ public Resource getResource(String location) {
 				return new FileSystemResource(cachedResource);
 			}
 		}
-		catch (URISyntaxException e) {
```

Comment:
do you think it is acceptable to throw a `
`ResourceNotResolvedException` for a potentially thrown `URISyntaxException`? (I guess it should never happen but ...